pilot/pkg/serviceregistry/kube/controller/util.go

```diff
@@ -137,21 +137,6 @@ func getPodServices(allServices []*v1.Service, pod *v1.Pod) []*v1.Service {
 	return services
 }
 
-func getPodsInService(allPods []*v1.Pod, svc *v1.Service) []*v1.Pod {
-	if svc.Spec.Selector == nil {
-		// services with nil selectors match nothing, not everything.
```

Comment:
I think we lost this

Reply:
It's here: https://github.com/istio/istio/pull/44134/files#diff-698573a8cf81b7bbd18b4e8d0e644ae539d40dfbefa1c95971ac85d0689f64cfR818

Reply:
oh thanks